<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="main_tab_widget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>70</y>
      <width>820</width>
      <height>440</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="dashboard_tab">
     <attribute name="title">
      <string>Dashboard</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="books_tab">
     <attribute name="title">
      <string>Books</string>
     </attribute>
     <widget class="QTabWidget" name="books_tab_widget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>820</width>
        <height>440</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="all_books_tab">
       <attribute name="title">
        <string>All Books</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="add_book_tab">
       <attribute name="title">
        <string>Add Book</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="edit_book_tab">
       <attribute name="title">
        <string>Edit Book</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="delete_book_tab">
       <attribute name="title">
        <string>Delete Book</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="issue_book_tab">
     <attribute name="title">
      <string>Issue Book</string>
     </attribute>
     <widget class="QTabWidget" name="books_tab_widget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>820</width>
        <height>440</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_10">
       <attribute name="title">
        <string>Lend Book</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_11">
       <attribute name="title">
        <string>Return Book</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="history_tab">
     <attribute name="title">
      <string>History</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="settings_tab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="dashboard_btn">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>70</y>
      <width>205</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Dashboard</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/dashboard.png</normaloff>:/icons/icons/dashboard.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="books_btn">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>156</y>
      <width>205</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Books</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/books.png</normaloff>:/icons/icons/books.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="history_btn">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>328</y>
      <width>205</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>History</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/history.png</normaloff>:/icons/icons/history.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="issue_book_btn">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>242</y>
      <width>205</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Issue Book</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/today.png</normaloff>:/icons/icons/today.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_btn">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>414</y>
      <width>205</width>
      <height>88</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Settings</string>
    </property>
    <property name="icon">
     <iconset resource="icons.qrc">
      <normaloff>:/icons/icons/settings.png</normaloff>:/icons/icons/settings.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <zorder>settings_btn</zorder>
   <zorder>history_btn</zorder>
   <zorder>issue_book_btn</zorder>
   <zorder>books_btn</zorder>
   <zorder>dashboard_btn</zorder>
   <zorder>main_tab_widget</zorder>
  </widget>
 </widget>
 <tabstops>
  <tabstop>main_tab_widget</tabstop>
  <tabstop>books_tab_widget</tabstop>
  <tabstop>issue_book_btn</tabstop>
  <tabstop>history_btn</tabstop>
  <tabstop>settings_btn</tabstop>
  <tabstop>dashboard_btn</tabstop>
  <tabstop>books_btn</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
